pkg/build/api/v1beta3/types.go

```diff
@@ -512,9 +512,33 @@ type BinaryBuildRequestOptions struct {
 type BuildLogOptions struct {
 	unversioned.TypeMeta
 
+	// The container for which to stream logs. Defaults to only container if there is one container in the pod.
+	Container string `json:"container,omitempty"`
```

Comment:
added descriptions. are these supposed to be picked up by swagger? because I can't see any diff when trying to update the spec